problemi/vectorial_operations/basic.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cstdlib>
+#include <iostream>
+
+int main() {
+  double x1;
+  double y1;
+  double z1;
+  std::cin >> x1 >> y1 >> z1;
+
+  if (std::cin.fail()) {
+    std::cerr << "One or more vector-1 input parameters are not valid\n";
+    return EXIT_FAILURE;
+  }
+
+  char op;
+  std::cin >> op;
+
+  if (std::cin.fail()) {
+    std::cerr << "Operator is not a valid char\n";
+    return EXIT_FAILURE;
+  }
+
+  double x2;
+  double y2;
+  double z2;
+  std::cin >> x2 >> y2 >> z2;
+
+  if (std::cin.fail()) {
+    std::cerr << "One or more vector-2 input parameters are not valid\n";
+    return EXIT_FAILURE;
+  }
+
+  double xr{};
```

Comment:
```suggestion
  double xr{0.};
```
E' meglio se imparano a inizializzare esplicitamente